build.cake

```diff
@@ -44,175 +45,73 @@ Task("Default")
-            {
-                var outputFileName = projectName + extension;
-                var outputFile = outputDirectory.CombineWithFilePath(outputFileName);
-                var artifactFile = artifactsBinNetStandard15Directory.CombineWithFilePath(outputFileName);
-                CopyFile(outputFile, artifactFile);
-            }
-        }
-    })
-    .Finally(() =>
+Task("Restore")
+    .IsDependentOn("Clean")
+    .Does(() => 
     {
-        GlobalAssemblyInfo.RestoreGlobalAssemblyInfoFile(Context, solutionDirectory);
+        DotNetCoreRestore(solutionFullPath);
     });
-
-Task("Test")
-    .IsDependentOn("TestWindows");
-
-Task("TestWindows")
-    .IsDependentOn("TestNet45")
-    .IsDependentOn("TestNetStandard15");
-
-Task("TestLinux")
-    .IsDependentOn("TestNetStandard15");
-
-Task("TestNet45")
-    .IsDependentOn("BuildNet45")
+Task("Build")
+    .IsDependentOn("Restore")
     .Does(() =>
     {
-        var testAssemblies = new List<FilePath>();
-        var testProjectNames = new []
-        {
-            "MongoDB.Bson.Tests",
-            "MongoDB.Driver.Core.Tests",
-            "MongoDB.Driver.Tests",
-            "MongoDB.Driver.GridFS.Tests",
-            "MongoDB.Driver.Legacy.Tests"
-        };
-        foreach (var testProjectName in testProjectNames)
-        {
-            var testAssembly = testsDirectory.CombineWithFilePath($"{testProjectName}/bin/{configuration}/{testProjectName}.dll");
-            testAssemblies.Add(testAssembly);
-        }
-        var testSettings = new XUnit2Settings
-        {
-            Parallelism = ParallelismOption.None,
-            ToolTimeout = TimeSpan.FromMinutes(30)
-        };
-        XUnit2(testAssemblies, testSettings);
+       var settings = new DotNetCoreBuildSettings { NoRestore = true, Configuration = configuration };
+       DotNetCoreBuild(solutionFullPath, settings);
     });
 
-Task("TestNetStandard15")
-    .IsDependentOn("BuildNetStandard15")
-    .Does(() =>
-    {
-        var testProjectNames = new []
-        {
-            "MongoDB.Bson.Tests.Dotnet",
-            "MongoDB.Driver.Core.Tests.Dotnet",
-            "MongoDB.Driver.Tests.Dotnet",
-            "MongoDB.Driver.GridFS.Tests.Dotnet",
-            "MongoDB.Driver.Legacy.Tests.Dotnet"
-        };
-        foreach (var testProjectName in testProjectNames)
-        {
-            var testProjectDirectory = testsDirectory.Combine(testProjectName);
-            DotNetCoreRestore(testProjectDirectory.ToString());
-            var testProjectFile = testProjectDirectory.CombineWithFilePath($"{testProjectName}.csproj");
-            var testSettings = new DotNetCoreTestSettings();
-            var xunitSettings = new XUnit2Settings
-            {
-                Parallelism = ParallelismOption.None,
-                ToolTimeout = TimeSpan.FromMinutes(30)
-            };
-            DotNetCoreTest(testSettings, testProjectFile, xunitSettings);
+Task("Test")
```

Comment:
I think this task should be called TestWindows.

It does exactly what the old TestWindows task did, namely run all tests against net45 and netstandard15.